apollo-gradle-plugin/src/main/java/com/apollographql/android/gradle/ApolloCodeGenInstallTask.java

```diff
@@ -7,46 +7,39 @@
 
 import org.gradle.api.Task;
 import org.gradle.api.specs.Spec;
-import org.gradle.api.tasks.OutputDirectory;
 
-import com.google.common.collect.Lists;
-
-import com.moowork.gradle.node.npm.NpmTask;
-import com.squareup.moshi.JsonReader;
+import com.moowork.gradle.node.npm.NpmInstallTask;
 import com.squareup.moshi.JsonWriter;
 import com.squareup.moshi.JsonAdapter;
 import com.squareup.moshi.Moshi;
 
 import okio.Okio;
 
-public class ApolloCodeGenInstallTask extends NpmTask {
+public class ApolloCodeGenInstallTask extends NpmInstallTask {
```

Comment:
extend NpmInstallTask directly and define the dependencies in the package.json file since we're creating one anyways.

This basically now runs `npm install` which would install packages under the dependencies object in the package.json file.